### PR TITLE
Created group card 🧑‍🤝‍🧑 

### LINKED ISSUE #25: Create group card
Card should always have the same height. If there is too much texts use `...`. Look for ellipsis. Card should be able to have dynamic width.

### PR DESCRIPTION
**Pull request by ThomasBernard0**

### CODE REVIEW

**Comment by RomainGuarinoni** [comment]
The group is taking th full width of the available space, is it norrmal ? 

On the figma mockup, it seems to have a `max-width`

image

File: components/GroupCard.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import Tag from '../components/Tag';
+import Link from 'next/link';
+
+type Props = {
+  title: string;
```
Comment (RomainGuarinoni) [inline components/GroupCard.tsx]: I think `name` instead of `title` is more explicit for a music group

File: components/GroupCard.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  subtitle?: string;
```
Comment (RomainGuarinoni) [inline components/GroupCard.tsx]: Same there, maybe use `nbMembers` or something like this to be more explicit. Why is it nullable ? Btw, by convention, Nullable props are given at the end of the object

File: components/GroupCard.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  genres,
+  href,
+}: Props) {
+  return (
+    <div className="border rounded-xl px-6 py-5">
+      <div className="flex items-center gap-3 border-b pb-4">
+        <div className="w-10 h-10 rounded-full bg-black"></div>
+        <div className="flex flex-col">
+          <span>{title}</span>
+          <span className="text-sm text-gray-500">{subtitle}</span>
```
Comment (RomainGuarinoni) [inline components/GroupCard.tsx]: I think the DB will return the number of members as a number like this : 

```ts
{
nbMembers : 4
}
```

so maybe the props should be : 

```ts
type Props = {
  title: string;
  nbMembers: number;
  description: string;
  genres: Array<string>;
  /**path to group page */
  href: string;
};
```

and the div like this : 

```tsx
<span className="text-sm text-gray-500">{nbMembers} membre{nbMembers>1 ? 's' : ''}</span>
```

What do @alexandrelam  think ?

**Comment by alexandrelam** [comment]
> The group is taking th full width of the available space, is it norrmal ?
> 
> On the figma mockup, it seems to have a `max-width`
> 
> image

Card component should not know its own size. His parent component should. It will be better when we'll have more cards and a better layout.

**Review by RomainGuarinoni — CHANGES_REQUESTED** [review]

**Review by alexandrelam — APPROVED** [review]
LGTM

**Pull request merged**